swim.go

```diff
@@ -16,8 +16,15 @@
 
 package swim
 
+import (
+	"time"
+)
+
 type Config struct {
 	MaxlocalCount int
+	T             int
```

Comment:
T and K without comment do not seem to be good expressions. Moreover, it is better to have a more understandable word than T and K with annotations.

Reply:
At first I wrote the variable names meaningfully, but I changed it to T and K because it is too long. Instead, I'll add a comment with the comment above the variable name.